src/components/views/rooms/RoomSublist.tsx

```diff
@@ -372,7 +372,7 @@ export default class RoomSublist extends React.Component<IProps, IState> {
     };
 
     private onTagSortChanged = async (sort: SortAlgorithm) => {
-        await RoomListStore.instance.setTagSorting(this.props.tagId, sort);
+        RoomListStore.instance.setTagSorting(this.props.tagId, sort);
```

Comment:
Is `RoomListStore.instance` actually the interface here, or is it `RoomListStoreClass`? It needs to be an interface else other parts of the app will be interacting with the concrete type.